aten/src/ATen/native/mkldnn/xpu/detail/Attr.h

```diff
@@ -371,4 +367,62 @@ class Attr {
   dnnl::post_ops dnnl_post_ops_;
 };
 
+static inline void construct_attr_for_unary(
+    const c10::string_view& unary_post_op,
+    const torch::List<std::optional<at::Scalar>>& unary_post_op_args,
+    const c10::string_view& unary_post_op_algorithm,
+    at::native::onednn::Attr& attr) {
+  if (unary_post_op == "relu") {
```

Comment:
Consider using switch statement rather then series if/else.

Reply:
hi @atalman Thanks for your suggestion! 
I did try modifying the code to use a switch statement, but since the argument string type rather than an integral type, it would lead to compilation errors. To make it work, we would need to add a mapping from string to a special enumeration class, which could complicate the utility function further. Would it be possible for us to keep the current implementation?